cinema/views.py

```diff
@@ -69,9 +67,31 @@ class MovieViewSet(
 ):
     queryset = Movie.objects.prefetch_related("genres", "actors")
     serializer_class = MovieSerializer
-    authentication_classes = (TokenAuthentication,)
     permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type=OpenApiTypes.STR,
+                description="Filter by movie title (e.g. &title=Alien)",
+            ),
+            OpenApiParameter(
+                "genres",
+                type={"type": "array", "items": {"type": "numbers"}},
```

Comment:
The type for the 'genres' and 'actors' parameters in the `extend_schema` decorator should be `OpenApiTypes.INT` instead of `numbers`. Ensure that the correct type is used for better API documentation.